### DISCUSSION TITLE
How to use simulation parameters like --warmup-insts and --maxinsts in SimpleBoard?

### DISCUSSION THREAD

**Discussion by NiJiazhe** [gem5/gem5#749] [Q&A] [answered]
**Describe the bug**
Hi sir:
I'm now trying to build the system on SimpleBoard to run spec2017.I use the `common.Options` parameter in the simulation,
```
from m5.util import addToPath
addToPath("../")
from common import Options
Options.addCommonOptions(parser)
Options.addSEOptions(parser)
.
.
.
# Lastly we run the simulation.
simulator = Simulator(board=board)
simulator.run()
```
but I get an error:

```
Exception: The banned module 'common.Options' has been included. Please do not use this in your simulations. Reason: The options provided by 'Options' are not compatible with the gem5 standard library.

At:
  src/python/gem5/simulate/simulator.py(441): run
  /home/ronnie/Projects/riscv-fs/gem5-fdp/configs/spec2017/fdp_spec2017.py(535): <module>
  src/python/m5/main.py(630): main
gem5 Simulator System.  https://www.gem5.org
gem5 is copyrighted software; use the --copyright option for details.

```

I am new to gem5. Are there any other simulation parameters that can be used on SimpleBoard, something like `--warmup-insts` and` --maxinsts`? Or where can find the relevant documentation?
Hope for your information!Thanks~

**Affects version**
gem5 v23.0.1.0

**Host Operating System**
Ubuntu 18.04

**Host ISA**
X86

**Comment by qqli579** [comment, marked as answer]
You need to add it yourself in fdp_spec2017.py.
